otter/grade/containers.py

```diff
@@ -104,30 +121,39 @@ def launch_containers(
     """
     pool = ThreadPoolExecutor(num_containers)
     futures = []
+    monitor_progress(result_queue, print_progress, f"Building image {base_image}:{tag}")
     image = build_image(ag_zip_path, base_image, tag, config)
 
     for subm_path in submission_paths:
-        futures += [pool.submit(
-            grade_submission,
-            submission_path=subm_path,
-            image=image,
-            # config=config,
-            **kwargs,
-        )]
-
-    # stop execution while containers are running
-    finished_futures = wait(futures)
-    scores = [f.result() for f in finished_futures[0]]
+        futures += [asyncio.wrap_future(pool.submit(
+            run_async_in_thread,
+            grade_submission(
+                submission_path=subm_path,
+                image=image,
+                # config=config,
```

Comment:
can you please remove this? ik you didn't add it but not sure why this is still here